mappings/net/minecraft/world/World.mapping

```diff
@@ -278,9 +281,13 @@ CLASS none/aiw net/minecraft/world/World
 		ARG 0 value
 	METHOD j getRainGradient (F)F
 		ARG 0 deltaTicks
+	METHOD j getSkyLightLevel (Lnone/cn;)I
```

Comment:
As far as I can tell, in `World` there are only functions for "all light" (sky + block) and "sky light". The block light is not retrieved by itself without adding the sky to it.